libbeat/dashboards/importer.go

```diff
@@ -245,7 +247,12 @@ func (imp Importer) downloadFile(url string, target string) (string, error) {
 func (imp Importer) ImportKibanaDir(dir string) error {
 	var err error
 
-	dir = path.Join(dir, imp.version)
+	versionPath := "default"
+	if imp.version.Major == 5 {
+		versionPath = "5x"
```

Comment:
@ruflin For when you are back from vacation, I think this should be `5.x` instead of `5x`. I'm going to make it `5.x` in a PR, but let me know if it was `5x` on purpose.

Reply:
That was definitively a typo.